simulator/risc_v/t/unit_test.cpp

```diff
@@ -216,3 +217,34 @@ TEST_CASE("RISCV RV64 orn")
     instr.execute();
     CHECK( instr.get_v_dst() == 0x4000'0000'0000'10ff);
 }
+
+TEST_CASE( "RISCV sro32")
+{
+    CHECK( RISCVInstr<uint32>( 0x20d65733).get_disasm() == "sro $a4, $a2, $a3");
+    RISCVInstr<uint32> test_32_1( "sro", 0);
+    test_32_1.set_v_src( 0x8000'c000, 0);
+    test_32_1.set_v_src( 0xf, 1);
+    test_32_1.execute();
+    CHECK( test_32_1.get_v_dst() == 0xffff'0001);
+
```

Comment:
I don't know why Visual Studio has a different output here.
One of guesses is - these literals are treated as signed, but not unsigned. Please use "u" or "ull" postfix.